XenAdmin/Wizards/NewSRWizard.cs

```diff
@@ -374,7 +384,7 @@ protected override void UpdateWizardContent(XenTabPage senderPage)
                 m_srWizardType.DeviceConfig = xenTabPageNfsIso.DeviceConfig;
                 SetCustomDescription(m_srWizardType, xenTabPageNfsIso.SrDescription);
             }
-            else if (senderPagetype == typeof(VHDoNFS))
+            else if (senderPagetype == typeof(VHDoNFS) || senderPagetype == typeof(VHDoNFSv2))

```

Comment:
Calling SetCustomDescription() for the new NFSv2 page is missing at 3 lines below